### PR TITLE
Create new file if.js, change .html button timeOfYear

### PR DESCRIPTION
**Pull request by veronaten**
Task 15, topic If-else

### CODE REVIEW

File: scripts/if.js [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
     return;
   }
 
-  const timeOfYear = year[userNumber - 1];
+  const timeOfYear = yearMonths[userNumber - 1];
 
-  alert(`It is ${timeOfYear}!`);
+  switch (userNumber) {
+    case 1:
+      alert(`It is ${timeOfYear}!`);
```
Comment (magicsd) [inline scripts/if.js]:
```javascript
alert(`It is ${yearMonths[0]}!`); 
```

and so on

File: scripts/if.js [hunk 2]
```diff
@@ -1,15 +1,44 @@
...
+    case 3:
+      alert(`It is ${timeOfYear}!`);
+    break;
+    case 4:
+      alert(`It is ${timeOfYear}!`);
+    break;
+  }
+}
+
+const months = {1: 'winter', 2: 'winter', 3: 'winter', 4: 'spring', 5: 'spring', 6: 'spring', 7: 'summer', 8: 'summer', 9: 'summer', 10: 'autumn', 11: 'autumn', 12: 'autumn'};
```
Comment (magicsd) [inline scripts/if.js]: Use existing array of seasons

Comment (veronaten) [reply scripts/if.js]: ok.

File: scripts/if.js [hunk 3]
```diff
@@ -1,44 +1,46 @@
...
-  alert(`Your time of year is ${result}.`);
+  if (userMonth == 1 && userMonth < 4) {
+    alert(`It is ${seasons[0]}`); 
+  } else if (userMonth >= 4 && userMonth < 7) {
+    alert(`It is ${seasons[1]}`);
+  } else if (userMonth >=7 && userMonth < 10) {
+    alert(`It is ${seasons[2]}`);
+  } else if (userMonth >= 10 && userMonth <= 12) {
+    alert(`It is ${seasons[3]}`);
+  }
```
Comment (magicsd) [inline scripts/if.js]: Switch-case please. Check spaces

File: scripts/if.js [hunk 4]
```diff
@@ -43,4 +43,22 @@ function getMonth() {
   } else if (userMonth >= 10 && userMonth <= 12) {
     alert(`It is ${seasons[3]}`);
   }
+}
+
+function checkNumber() {
+  const userNumber = prompt('Enter 5 digits', 12345) || '0';
```
Comment (magicsd) [inline scripts/if.js:66]:
```javascript
  const userNumber = prompt('Enter 5 digits', '12345')
  ```
  You've already put fallback here

Comment (veronaten) [reply scripts/if.js:66]: Yes, thank you)

Comment (magicsd) [reply scripts/if.js:66]: Or not?

File: scripts/if.js [hunk 5]
```diff
@@ -34,19 +34,32 @@ function getMonth() {
...
+    case 8:
+    case 9:
+      alert(`It is ${seasons[2]}`);
+    break;
+    case 10:
+    case 11:
+    case 12:
+      alert(`It is ${seasons[3]}`);
+    break;
+  }
```
Comment (magicsd) [inline scripts/if.js]: Call `alert` only one time

File: scripts/if.js (hunk 4 above)
Comment (veronaten) [reply scripts/if.js:66]: Not, there is a difference with || "0" and without || "0".

**Review by magicsd — APPROVED** [review]

**Pull request merged**